src/Main.elm

```diff
@@ -1361,19 +1333,8 @@ encodePresence presence =
         Optional ->
             Json.Encode.string "Optional"
 
-        SystemRequired sys ->
-            Json.Encode.object
-                [ ( "type", Json.Encode.string "SystemRequired" )
-                , ( "name", Json.Encode.string sys.name )
-                , ( "description", Json.Encode.string sys.description )
-                ]
-
-        SystemOptional sys ->
-            Json.Encode.object
-                [ ( "type", Json.Encode.string "SystemOptional" )
-                , ( "name", Json.Encode.string sys.name )
-                , ( "description", Json.Encode.string sys.description )
-                ]
+        System ->
+            Json.Encode.string "System"
 
```

Comment:
`System` is simply encoded as a `"System"` string instead of a json object; we carry no extra attributes payload